app/controllers/api/v1/likes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Api::V1::LikesController < ApplicationController
+  before_action :find_photo, :authorized, :find_like
+
+  def destroy
+    if !Likes::AlreadyLikedService.new(like: @like, photo: @photo, user: @user).call || @like.nil?
```

Comment:
Controller contains logic. Better to use one service Likes::Pusher and check liked or not there